src/config.rs

```diff
@@ -163,20 +163,43 @@ impl Config {
 
 	/// Update the command-line arguments based on configuration.
 	pub fn update_args(&self, mut args: Args) -> Args {
-		args.armor = self.gpg.armor;
-		args.splash = self.general.splash;
+		let default_color = Color::from("gray");
+		let default_style = Style::from_str("plain", true).unwrap_or_default();
+		let default_file_exp = String::from("xplr");
```

Comment:
Can you use constants for these values?

Reply:
Like do you want lazy_static/and get those Color + Style variables themselves made constant? or do you want just the &str values made constant and put into the runtime functions there?

Reply:
I would assume the latter actually now that I think about it.